src/lib.rs

```diff
@@ -492,32 +464,39 @@ impl HdfsMultipartWriter {
     ) -> Self {
         let (sender, receiver) = mpsc::unbounded_channel();
 
```

Comment:
// how/where is HdfsMultipartWriter used. who closes the file?
// We should add tests for this